src/md/src/InstallationSteps/DeployReportPortalDocker.md

```diff
@@ -101,30 +98,26 @@ Where:
 
 You can get the host IP address by using the following docker commands:  
 
-> If you run Docker on macOS or Windows with Docker for Mac, Docker for Windows, or Docker Toolbox  
+> If you run Docker on macOS or Windows with Docker for Mac, Docker for Windows  
 
-```shell
-docker-machine ip default
+```Bash
+docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' CONTAINER_ID_OR_NAME
```

Comment:
Please escape reserved variables definition for documentation-assembly as it was done [here](https://github.com/reportportal/documentation/pull/344/files).